pandas/core/frame.py

```diff
@@ -2277,7 +2277,7 @@ def to_markdown(
         result = tabulate.tabulate(self, **kwargs)
         if buf is None:
             return result
-        buf, _, _, should_close = get_filepath_or_buffer(
+        buf, _, _, should_close, mode = get_filepath_or_buffer(
```

Comment:
I will try to make a test for `to_markdown` to cover this line

Reply:
should already be covered by the tests in pandas/tests/io/formats/test_to_markdown.py.

I don't know why the coverage test complains about that.